README.md

```diff
@@ -110,6 +110,47 @@ You can use an in-memory SQLite driver by adding the `-M` option.
 You can choose to include helper files. This is not enabled by default, but you can override it with the `--helpers (-H)` option.
 The `Illuminate/Support/helpers.php` is already set up, but you can add/remove your own files in the config file.
 
+#### Cooperation with release builds
+Create a file `app/Console/Commands/IdeHelperDoIt.php`:
+```php
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\Artisan;
+
+class IdeHelperDoIt extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'ide-helper:do-it';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Run barryvdh/laravel-ide-helper';
+
+    public function handle()
+    {
+        if ('local' === config('app.env', 'production')) {
+            Artisan::call('ide-helper:generate');
+            Artisan::call('ide-helper:meta');
+            Artisan::call('ide-helper:models', ['--nowrite' => true]);
+        }
+    }
+}
+```
+
+In `composer.json` you can then add `"@php artisan ide-helper:do-it"` that will gracefully work indepent of `--no-dev` composer installs.
```

Comment:
Maybe it should be more explicit _where_ to add this, as they're multiple keys possible in `scripts`, i.e. mention `post-update-cmd` ?